kolibri/core/analytics/pskolibri/__init__.py

```diff
@@ -0,0 +1,292 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import os
+import sys
+import time
+
+from kolibri.core.analytics.pskolibri.common import LINUX
+from kolibri.core.analytics.pskolibri.common import memoize_when_activated
+from kolibri.core.analytics.pskolibri.common import NoSuchProcess
+from kolibri.core.analytics.pskolibri.common import PY3
+from kolibri.core.analytics.pskolibri.common import WINDOWS
+# This module is heavily based on the psutil module
```

Comment:
Could this be at the top instead, as a docstring? That would make it more visible.

The intention is of course both that we give attribution and mention licenses.. but _also_ that we, ourselves, can make future decisions about this codebase based on its vendored origin.. and not somehow that we wrote this ourselves :)

Reply:
@benjaoming 
 I am not sure of how you want the text to be wording. Please, feel free to change it yourself. I am totally in favour of mentioning others work.